src/API/WorkController.ts

```diff
@@ -3,10 +3,10 @@ import { inject, injectable } from 'inversify'
 import { Collection, Db } from 'mongodb'
 import * as Pino from 'pino'
 
+import 'Extensions/Array'
```

Comment:
Add imports with side effects to the root of the application whenever possible:

https://github.com/poetapp/node/blob/e75ba844c520b09a4532ee55a1e98ea374d19d58/src/index.ts#L1-L10